src/gui/editors/PianoRoll.cpp

```diff
@@ -766,6 +770,79 @@ void PianoRoll::fitNoteLengths(bool fill)
 }
 
 
+
+bool fillGapsBetweenNotesInVector(
```

Comment:
```suggestion
static bool fillGapsBetweenNotesInVector(
```
This could be static since it's only used in this translation unit.